### ISSUE TITLE
Tutorials

### ISSUE THREAD

**Issue by imachug** [krixano/ZeroNet-Dev-Center#28]
I just wrote a few tutorials before I found your site. I cannot manually add tutorials, so take a look at zip then. [The truth about ZeroNet.zip](https://github.com/krixano/ZeroNet-Dev-Center/files/1334487/The.truth.about.ZeroNet.zip)

**Comment by krixano** [comment]
I will take a look at them eventually. Sorry, I've been busy with college. Right now my first priority is to get my other zite (ZeroMedium) done and released.

**Comment by anoadragon453** [comment]
@imachug These tutorial are amazing and so clear! @krixano please consider adding these to the site in some form ASAP. Such good content.

Also @imachug it seems that some of the HTML files in the downloads folder are blank. Is there supposed to be content in those? Thanks again!

**Comment by krixano** [comment]
I will be working on the site again soon (I get off of college for the semester on December 15th). Just be patient please. I do intend to integrate these tutorials into the ones that I will be written (with permission from imachug).

**Comment by choutigi** [comment]
@imachug i agree with @anoadragon453 i read your tutorial and got a site up in about 20 minutes.

thank you for your effort!

**Comment by imachug** [comment]
> Also @imachug it seems that some of the HTML files in the downloads folder are blank.

I wanted to bring up a site but noticed ZeroNet Dev Center and thought that's a better place for them. Those HTMLs should just link to ZIPs.

**Comment by krixano** [comment]
In the future, I'm planning on redoing the whole zite. I'm still unsure on how I want the whole zite organized, but once I figure that out, I'll start redoing the whole zite. But that won't be for a while, so I'm just going to focus on writing at least a few more tutorials right now.

**Comment by krixano** [comment]
@imachug I see you included your ZeroPage library in the zip folder. I wanted to see if you made any changes to that after you created this zip file or not.

**Comment by imachug** [comment]
Yeah, probably made. There is a repo for them on GitHub: https://github.com/imachug/ZeroLib. You may want to add a notice about it.

**Comment by filips123** [comment]
I created a template for ZeroNet sites with support for NPM dependencies, SASS, Gulp, Browserify and automatic deployments with Travis CI. It's available on [GitHub](https://github.com/filips123/ZeroTemplate) and [GitCenter](http://127.0.0.1:43110/1Aob9QXahTnn6iBriSSQ6qg8qXkHdkaX2x).

Could you add it to those tutorials?

**Comment by anoadragon453** [comment]
@filips123 Wonderful!

I'm afraid the developer of this zite has moved on from ZeroNet though.

Perhaps you could post it somewhere else (or fork the zite)?

I do hope it reaches a lot of people however. ZeroNet desperately needs more zite creators.
